### PR TITLE
feat(support-information): Create support-info.md

### PR DESCRIPTION
**Pull request by HeyRoach**

### CODE REVIEW

File: src/support/support-info.md [hunk 1]
```diff
@@ -0,0 +1 @@
+For more information to write support@cloudstack.com
```
Comment (HeyRoach) [inline src/support/support-info.md]: it is just for testing.
After testing i will remove this file

File: src/app/shared/components/support-information/support-information.component.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<ng-container *ngIf="isShow">
+  <div class="support-information">
+    <h4>{{ 'ERRORS.SUPPORT_INFORMATION' | translate }}</h4>
+    <button
```
Comment (ksendart) [inline src/app/shared/components/support-information/support-information.component.html]: Why do not you use <mat-expansion-panel>? (https://material.angular.io/components/expansion/overview)
I suppose that it looks better, and we do not need any CSS for it. It is just a comment, you can use your own solution :)

File: src/app/shared/components/support-information/support-information.component.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      })
+      .subscribe(supportText => {
+        this.supportInformation = supportText;
+      });
+  }
+
+  public readFile() {
+    return this.getPath()
+      .switchMap(responsePath => this.http.get(responsePath))
+      .catch(() => this.http.get(this.defaultLangPath))
```
Comment (zolotyx) [inline src/app/shared/components/support-information/support-information.component.ts]: Please think if we need to cache that responses

File: src/app/shared/components/support-information/support-information.component.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { Http } from '@angular/http';
+import { Converter } from 'showdown';
+import { UserTagService } from '../../services/tags/user-tag.service';
+import { Observable } from 'rxjs/Observable';
+
+@Component({
```
Comment (zolotyx) [inline src/app/shared/components/support-information/support-information.component.ts]: Maybe a separate module should be created for this component and the functionality described in this ticket

File: src/app/support-information/support-information.component.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        const converter = new Converter();
+        return converter.makeHtml(text);
+      })
+      .subscribe(supportText => {
+        this.supportInformation = supportText;
+      });
+  }
+
+  public readFile() {
+    const cachedRequest = this.requestCache.get({});
```
Comment (zolotyx) [inline src/app/support-information/support-information.component.ts]: 1 - we need to cache responses, not requests
2 - are you sure that your code works correctly if we change current language?

Comment (zolotyx) [inline src/app/support-information/support-information.component.ts]: can we change the signature of cache.get not to pass empty object if not required?

File: src/app/support-information/support-information.component.ts [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+})
+export class SupportInformationComponent implements OnInit {
+  protected supportInformation: string;
+  protected defaultPath = 'support/support-info';
+  protected defaultLangPath = `${this.defaultPath}.${'md'}`;
+  protected isShow = true;
+  protected requestCache: Cache<Observable<FormattedResponse<string>>>;
+
+  constructor(
+    private http: Http,
```
Comment (zolotyx) [inline src/app/support-information/support-information.component.ts]: should it be Http of HttpClient (please check agreement dialog and do the changes if required)

File: src/app/support-information/support-information.module.ts [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import { SupportInformationComponent } from './support-information.component';
```
Comment (zolotyx) [inline src/app/support-information/support-information.module.ts]: Please rename module directory to **support**

File: src/app/shared/services/cache.ts [hunk 8]
```diff
@@ -11,8 +11,8 @@ export interface CacheEntry<T> {
 export class Cache<T> implements ICache<T> {
   private cache: { [key: string]: T; } = {};
 
-  public get(params: {}): T {
-    const key = JSON.stringify(params);
+  public get(params?: {}): T {
+    const key = JSON.stringify(params || {});
```
Comment (zolotyx) [inline src/app/shared/services/cache.ts]: you can set default value in method declaration

**Pull request closed without merging**